server/src/main/java/io/crate/execution/engine/aggregation/impl/ArbitraryAggregation.java

```diff
@@ -130,6 +131,7 @@ public Object terminatePartial(RamAccounting ramAccounting, Object state) {
         return state;
     }
 
+    @VisibleForTesting
```

Comment:
Please remove this again. We usually use this annotation to indicate that a method has the public or package private visibility only to make it available in tests.  This is not the case here - this is public API.